lib/cacheconfiguration.h

```diff
@@ -41,6 +41,9 @@
 #ifdef HAS_MLITE
 #include <mgconfitem.h>
 #endif
+#ifdef HAS_GSETTINGS
```

Comment:
Suppose QGSETTINGS would be more precise naming for this. On GSettings I first assumed this was about glib.

Reply:
It is about GSETTINGS from glib! (via gsettings-qt).

Reply:
Note that Ubuntu Touch / Lomiri is a Qt based shell but using the glib2.0 gsettings backend for storing configuration data.